pokeapp/src/components/PokemonCard.js

```diff
@@ -60,6 +55,15 @@ const PokemonCard = ({
     borderRadius: `10px`,
   };
 
+  const hoverMe = () => {
+    setActive(true);
+    setFirstHover(true);
+  };
+
+  const leaveHover = () => {
+    setActive(false);
+  };
```

Comment:
You could do this in one-line to look saucy if you are just returning one thing in the function: `const leaveHover = () => (setActive(false));`

Reply:
Thank you sir good looking out!